src/pages/Detail/Details.js

```diff
@@ -0,0 +1,82 @@
+import CloseButton from 'react-bootstrap/CloseButton';
+import ListGroup from 'react-bootstrap/ListGroup';
+import React, { useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import './details.css';
+import { allCoins } from '../../redux/home-reducer';
+
+function Details() {
+  const getName = useParams();
+  const { name } = getName;
+  const Coins = useSelector((state) => state.coin);
+  const send = useDispatch();
+  useEffect(() => {
+    send(allCoins());
+  }, [send]);
+
+  const filtere = Coins.filter((item) => item.name === name);
+  return (
+    <div className="detail-container">
+      <div className="detail-wrapper">
+        <Link to="/" type="button"><CloseButton /></Link>
+        {
+          filtere.map((item) => (
+            <div key={item.id}>
+              <div className="header">
+                <img src={item.icon} alt={item.name} />
+                <h1>{item.name}</h1>
+
+              </div>
+              <ListGroup as="ol" numbered className="items-list">
+                <ListGroup.Item as="li">
+                  Rank :
+                  {item.rank}
+                </ListGroup.Item>
+                <ListGroup.Item as="li">
+                  Price :
+                  {Math.round(item.price * 100) / 100 }
+                </ListGroup.Item>
+                <ListGroup.Item as="li">
+                  Price in BTC :
+                  {Math.round(item.priceBtc * 100) / 100 }
```

Comment:
![image](https://user-images.githubusercontent.com/79058364/217826455-b06fb502-f6f4-4064-8c34-ed92221dc9f1.png)

- [ ] According to the project requirements  🙃  Let's make sure the Details page stick to the original design as much as possible 🎨 🖌️  ( You need to change your element flex Layout, background color, width, size, .... and add missing icons)

Your Implementation | Original Design
:---:|:---:
![image](https://user-images.githubusercontent.com/79058364/218007627-a4e9c2a2-4b7e-454d-9de7-607bb548bc8f.png)| ![image](https://user-images.githubusercontent.com/79058364/217828693-b49d57ec-1cb0-48d3-83e8-743823b60f19.png)